tests/sqlparser_clickhouse.rs

```diff
@@ -220,6 +220,142 @@ fn parse_create_table() {
     );
 }
 
+#[test]
```

Comment:
can we add a dedicated (e.g. parse_clickhouse_data_types) test that enumerates the new datatypes added by the PR? so that they have coverage and are doing what we intended them to

Reply:
I added a test `parse_clickhouse_data_types` with parsing a query included all new simple data types introduced in this PR.